### DISCUSSION TITLE
there always one mistake！！！

### DISCUSSION THREAD

**Discussion by zyq-20000901** [community/community#50439] [GitHub Education]
our application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile]with your full name as it appears in your academic affiliation document plus a short bio.
Have you completed your [GitHub user profile]with all your relevant information, such as your full name as it appears in your image and a short bio?

It's really a waste of my time!
has anybody can help me?
I have tried my best to make it.
using my full name, translate my report...
upload and taking photo are all no success.
Have any staff can help me? By manual verification.

**Comment by aviiciii** [comment]
Same problem for me!
